src/VisualizerModule.py

```diff
@@ -175,7 +175,7 @@ def Draw_point(self, canvas, x, y, color):
             x_transform - 5, y_transform - 5, x_transform + 5, y_transform + 5, fill=color, outline=color
         )
 
-    def Input_predict(self):
+    def input_predict(self):
```

Comment:
Function naming convention is to use nouns for the last word. Maybe change it to input_prediction.